docs/developer-guide/create-control-panel.md

```diff
@@ -0,0 +1,390 @@
+---
+myst:
+  html_meta:
+    "description": "How to create a control panel"
+    "property=og:description": "How to create a control panel"
+    "property=og:title": "Create a control panel"
+    "keywords": "Plone, create, control panel, plonecli, registry"
+---
+
+(backend-controlpanels-label)=
+
+# Create a control panel
+
+There are two approaches to create a control panel for your Plone add-on:
+
+-   [`plonecli`](https://pypi.org/project/plonecli/)
+-   manual
+
+
+## `plonecli`
+
+To add a control panel to your add-on, you can use [`plonecli`](https://pypi.org/project/plonecli/) as follows.
+
+```shell
+plonecli add controlpanel
+```
+
+This creates the control panel Python file in the control panel's folder where you can define your control panel schema fields.
```

Comment:
Where should the reader go from here? I'm lost. Help! In other words, write documentation as if you didn't have years of experience with Plone.

Reply:
Okay, after running the `plonecli` command to create controlpanel. It creates control panel with a basic field (schema.TextLine) for the CLassic and Volto both, like an angle 😇😎

So, ig the reader will now be required to know the schemas, then `Fieldset` if needed to group fields together

Reply:
In that case, we need to direct the reader to those parts of the documentation.

And now that I think about it, we should have only one method. plonecli automatically performs all the steps in the manual process for the reader, so it's silly to say that there are two processes. The steps are identical, and the only difference is automation of performing them. In any case, the reader will need to modify their add-on's schema, as plonecli has no way to read the developer's mind. Yet.

Reply:
IMO, we can simply add a note that plonecli does everything done in the manual process... since, the manual process exposes developers to some minor technicalities of the thing